src/App.js

```diff
@@ -1,8 +1,10 @@
 import {useState} from 'react';
+import { Routes, Route} from 'react-router-dom';
 import styled from 'styled-components';
 
-import ChairCard from './component/ChairCard/ChairCard';
+import ChairMapping from './component/ChairMapping/ChairMapping';
```

Comment:
I suggest to find a better name for your component... Maybe something like `ChairsList` 🤔